Components/DateComponent/DateField.js

```diff
@@ -0,0 +1,56 @@
+import DateTimePicker from '@react-native-community/datetimepicker';
+import {View,Button,Text} from 'react-native'
+import React,{useState,useEffect} from 'react'
+
+const DateField = (props)=>{
+    const [date, setDate] = useState(new Date(1598051730000));
+    const [mode, setMode] = useState('date');
+    const [show, setShow] = useState(false);
+    const onChange = async (event, selectedDate) => {
+        const currentDate = selectedDate;
+        setShow(false);
+        await setDate(currentDate);
+      };
+      const showMode = (currentMode) => {
+        if (Platform.OS === 'android') {
+          setShow(false);
+          // for iOS, add a button that closes the picker
+        }
+        setMode(currentMode);
+      };
+      const showDatepicker = () => {
+        showMode('date');
+        setShow(true)
+      };
+      useEffect(() => {
+        let holderDate = date
+        dateString = date_TO_String(holderDate)
+        props.updateDate(dateString)
+      }, [date])
+      return(
+        <View>
+            <Button onPress={showDatepicker} title="Select Date of Birth" />
+                <Text style={{alignItems:'center',textAlign:'center'}}>Selected: {`${date.getFullYear()}-${date.getMonth()+1}-${date.getDate()}`}</Text>
+                {show && (
+                    <DateTimePicker
+                    testID="dateTimePicker"
+                    value={date}
+                    mode={mode}
+                    onChange={onChange}
+                    />
+                )}
+        </View>
+      )
+}
+
+function date_TO_String(date_Object) {
+    // get the year, month, date, hours, and minutes seprately and append to the string.
+    var date_String = date_Object.getFullYear() +
+       "-" +
+       (date_Object.getMonth() + 1) +
```

Comment:
Why are we incrementing on the months here?

Reply:
Javascript getMonth function returns 0 in case u're referencing months normally in a typical array, ex. 
`var months = ["January", "February", "March", "April", "May", "June", "July",
         "August", "September", "October", "November", "December"]`;
so returning 0 would be direct access for the months instead of decrementing it if it starts from 0
as referred to in here https://stackoverflow.com/questions/15799514/why-does-javascript-getmonth-count-from-0-and-getdate-count-from-1